.gitlab/deploy_packages/nix.yml

```diff
@@ -146,17 +146,29 @@ deploy_packages_dogstatsd_suse_rpm-x64-7:
   variables:
     PACKAGE_ARCH: x86_64
 
-deploy_packages_dmg-x64-a7:
+.deploy_packages_dmg:
```

Comment:
To be consistent with other deploy job templates, we may want to move this to https://github.com/DataDog/datadog-agent/blob/regis.desgroppes/macos-aarch64/.gitlab/deploy_packages/deploy_common.yml.